README.md

```diff
@@ -1 +1,20 @@
-# RankingNumberBot
\ No newline at end of file
+# RankingNumberBot
+
+**Running bot locally**
+1. create a local postgresql database
+2. set up the database by running the stuff in the sql_setup.sql file using your favourite database management software
+3. copy .env.example into a .env file
+4. set environment variables POSTGRESS_USER, DB_PASSWORD and DB_NAME
+   - if desired you can set the DATABASE_URL environment variable to the correct connection string
+   - keep SSL_MODE at false, unless you need it
+       - if SSL_MODE is missing from .env it will default to using SSL for heroku
+5. create a discord app on https://discord.com/developers and create a bot user too
+6. set the DISCORD_SECRET environment variable to the bot token (not the application token)
+7. set the GUILD_IDS environment variable to the id of your server for faster development
+8. change the configuration in constants.py (optional, only needed for very few features/commands)
+9. install dependencies usinhg "pip install -r requirements.txt"
+10. run main.py and enjoy
```

Comment:
```suggestion
1. Create a local PostgreSQL database
2. Set up the database by running the commands in the `sql_setup.sql` file using your favourite database management software
3. Copy `.env.example` into a `.env` file
4. Set environment variables `DB_USER`, `DB_PASSWORD` and `DB_NAME`
   - If desired, you can set the `DATABASE_URL` environment variable to the full connection string instead
   - Keep `SSL_MODE` set to false when running the bot locally, unless you need it
       - If `SSL_MODE` is missing from `.env`, it will default to using SSL
5. Create Discord application at https://discord.com/developers and add a bot user
6. Set the `DISCORD_SECRET` environment variable to the bot token (not the application token)
7. Set the `GUILD_IDS` environment variable to the id of your server to force Discord to register the commands more quickly
8. Change the configuration in `constants.py` (optional, only needed for very few features/commands such as listening to message reactions)
9. Install dependencies using `pip install -r requirements.txt`
10. Run main.py and enjoy
```

Reply:
not the full connection string, as in its without the ?sslmode=require
that is set depending on the SSL_MODE env

Reply:
That's a good point, I changed it to make it clearer